twingate/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c88bd253-18da-4224-af14-7854ce8ae6ed",
+  "app_id": "twingate",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Twingate provides a modern, Zero Trust alternative to corporate VPNs",
+    "title": "Twingate",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Category::Network",
+      "Category::Security",
+      "Category::Cloud",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Google Cloud"
+    ]
+  },
+  "author": {
+    "support_email": "support@twingate.com",
+    "homepage": "https://github.com/DataDog/integrations-extras",
+    "sales_email": "sales@twingate.com",
+    "name": "unknown"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "Twingate",
+      "configuration": {},
+      "events": {
+        "creates_events": true
```

Comment:
The README file mentions this integration does not collect events. If this doesn't collect events, set this to false instead of true